libs/database/src/repositories/inspection-application.repository.ts

```
@@ -376,7 +376,9 @@ export class InspectionApplicationRepository {
         skip: offset,
         take: limit,
         orderBy: {
-          createdAt: 'desc',
+          inspectorSlot:{
```

ESLint

[error] Insert `·`

(prettier/prettier)